lean/commands/logs.py

```diff
@@ -0,0 +1,77 @@
+# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean CLI v1.0. Copyright 2021 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+from pathlib import Path
+from lean.click import LeanCommand,PathParameter
+from lean.container import container
+
+logger = container.logger()
+
+
+@click.command(cls=LeanCommand)
+@click.option("--live",'mode',flag_value="live",help="get latest live log")
+@click.option("--backtest",'mode',flag_value="backtests",help="get latest backtests log")
+@click.option("--optimization",'mode',flag_value="optimizations",help="get latest optimization log")
```

Comment:
These are mutually exclusive, should check if the user only provides one of them and abort if multiple are given.